paper/paper.md

```diff
@@ -136,13 +136,25 @@ authentication) as well as processing functions for unpacking archives (zip or
 tar) and decompressing files (gzip, lzma, and bzip2).
 
 To the best of the authors' awareness, the only other Python software with some
-overlapping functionality is [Intake](https://github.com/intake/intake).
-While Intake is powerful and can be used to manage large data archives,
-we argue that Pooch has a simpler setup and meets the
-specific needs of scientific software authors and individual scientists.
-For example, Pooch does not require users to change their data loading code to
-fit into a plug-in structure, instead only providing the file path for the
-user.
+overlapping functionality are [Intake](https://github.com/intake/intake) and
+[fsspec](https://github.com/intake/filesystem_spec) (which is the backend for Intake).
+The fsspec library provides a unified interface for defining file systems and
+opening files, regardless of where the files are located (local system,
+HTTPS/FTP servers, Amazon S3, Google Cloud Storage, etc).
+fsspec implements similar download and caching functionality to
+the one in Pooch, but has a wider range of download methods available.
+In the future, fsspec could be used as a backend to expand Pooch's download
+capabilities beyond HTTPS and FTP.
+Intake manages data catalogues (with download locations and extensive
+metadata), data download and caching (using fsspec), data loading,
```

Comment:
(part of the confusion arises because Intake also has its own caching layer outside of fsspec, which I intend to deprecate)

Reply:
:+1: thanks for catching that. Would it be more accurate to remove the `(using fsspec)` part for now?

Reply:
You already say that Intake uses fsspec for its file handling, so probably there's no need to repeat it here - i.e., yes, can remove the parentheses. The "old" caching mechanism in Intake is not something I want to promote any more, since the fsspec one is far simpler and easier to understand.